src/middlewared/middlewared/plugins/unauthorized_reboot_alert.py

```diff
@@ -16,7 +15,9 @@ class UnauthorizedRebootAlertService(Service):
 
 async def setup(middleware):
     if os.path.exists(SENTINEL_PATH):
-        hostname = socket.gethostname()
+        gc = await middleware.call('datastore.config', 'network.globalconfiguration')
+
+        hostname = gc['gc_hostname']
```

Comment:
gethostname will get a FQDN with hostname.domain, perhaps we need to add `gc_domain` to keep it the same?